src/content/docs/fr/reference/configuration-reference.mdx

```diff
@@ -0,0 +1,1711 @@
+
+Pour éviter les incohérences avec le comportement des barres obliques finales en développement, vous pouvez restreindre l'[option `trailingSlash`](#trailingslash) à `'always'` ou `'never'` selon votre format de build :
+- `directory` - Définissez `trailingSlash: 'always'`
+- `file` - Définissez `trailingSlash: 'never'`
+
+### build.client
+
+<p>
+
+**Type :** `string`<br />
+**Par défaut :** `'./dist/client'`
+</p>
+
+Contrôle le répertoire de sortie de vos fichiers CSS et JavaScript, côté client, avec `output: 'server'` ou `output: 'hybrid'` uniquement.
+`outDir` contrôle l'endroit où le code est construit.
+
+Cette valeur est relative à `outDir`.
+
+```js
+{
+  output: 'server', // ou 'hybrid'
+  build: {
+    client: './client'
+  }
+}
+```
+
+### build.server
+
+<p>
+
+**Type :** `string`<br />
+**Par défaut :** `'./dist/server'`
+</p>
+
+Contrôle le répertoire de sortie du JavaScript côté serveur lors de la construction en mode SSR.
+
+Cette valeur est relative à `outDir`.
+
+```js
+{
+  build: {
+    server: './server'
+  }
+}
+```
+
+### build.assets
+
+<p>
+
+**Type :** `string`<br />
+**Par défaut :** `'_astro'`<br />
+<Since v="2.0.0" />
+</p>
+
+Spécifie le répertoire dans la sortie de construction où doivent résider les ressources générées par Astro (JS et CSS regroupés par exemple).
+
+```js
+{
+  build: {
+    assets: '_custom'
+  }
+}
+```
+**Voir aussi :**
+- outDir
+
+### build.assetsPrefix
+
+<p>
+
+**Type :** `string | Record.<string, string>`<br />
+**Par défaut :** `undefined`<br />
+<Since v="2.2.0" />
+</p>
+
+Spécifie le préfixe des liens des ressources générées par Astro. Cela peut être utilisé si les ressources sont servies à partir d'un domaine différent de celui du site actuel.
+
+Cela nécessite de télécharger les ressources de votre dossier local `./dist/_astro` vers un dossier `/_astro/` correspondant sur le domaine distant.
+Pour renommer le chemin `_astro`, spécifiez un nouveau répertoire dans `build.assets`.
+
+Pour récupérer toutes les ressources téléchargées sur le même domaine (par exemple `https://cdn.exemple.com/_astro/...`), définissez `assetsPrefix` sur le domaine racine sous forme de chaîne de caractères (quelle que soit votre configuration `de base`) :
+
+```js
+{
+  build: {
+    assetsPrefix: 'https://cdn.exemple.com'
+  }
+}
+```
+
+**Ajouté dans :** `astro@4.5.0`
+
+Vous pouvez également transmettre un objet à `assetsPrefix` pour spécifier un domaine différent pour chaque type de fichier.
+Dans ce cas, une propriété `fallback` est requise et sera utilisée par défaut pour tous les autres fichiers.
+
+```js
+{
+  build: {
+    assetsPrefix: {
+      'js': 'https://js.cdn.exemple.com',
+      'mjs': 'https://js.cdn.exemple.com',
+      'css': 'https://css.cdn.exemple.com',
+      'fallback': 'https://cdn.exemple.com'
+    }
+  }
+}
+```
```

Comment:
```suggestion
  build: {
    assetsPrefix: 'https://cdn.example.com'
  }
}
```

**Ajouté dans :** `astro@4.5.0`

Vous pouvez également transmettre un objet à `assetsPrefix` pour spécifier un domaine différent pour chaque type de fichier.
Dans ce cas, une propriété `fallback` est requise et sera utilisée par défaut pour tous les autres fichiers.

```js
{
  build: {
    assetsPrefix: {
      'js': 'https://js.cdn.example.com',
      'mjs': 'https://js.cdn.example.com',
      'css': 'https://css.cdn.example.com',
      'fallback': 'https://cdn.example.com'
    }
  }
}
```
```

Reply:
The formatting of the suggestion was broken, so I updated manually (with credits 😉 ).